packages/pieces/community/captain-data/src/lib/actions/launch-workflow.ts

```diff
@@ -0,0 +1,36 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { httpClient, HttpMethod } from '@activepieces/pieces-common';
+import {
+  CAPTAIN_DATA_BASE_URL,
+  captainDataAuth,
+  CaptainDataAuthType,
+} from '../..';
+import { workflowProp } from '../common';
+
+export const launchWorkflow = createAction({
+  // auth: check https://www.activepieces.com/docs/developers/piece-reference/authentication,
+  name: 'launchWorkflow',
+  displayName: 'Launch a workflow',
+  description: '',
+  auth: captainDataAuth,
+  props: {
+    workflow: workflowProp,
+    payload: Property.Json({
+      displayName: 'Payload',
+      description:
+        'See https://docs.captaindata.co/#36e905b6-3a31-4bcd-8c6f-0eb6093b5a8a for details',
+      required: true,
+    }),
```

Comment:
In theory yes but it would require more work to generate the dynamic props - I'd rather do it in a second stage (particularly since I'm myself not very familiar with Captain Data)
My specific use case here is to prevent my users from using a generic HTTP pieces with unsafe API keys in headers 😬

Reply:
We can add this action in the second stage, as it will make it easier for the user to set up the workflow configuration from dynamic properties. What do you think?

Reply:
Yes i will add it later - by second stage I meant "later PR" 😀

Reply:
Yes, please add this action in the next PR. Let's merge it after you remove it.

Reply:
@kishanprmr I've spent a few hours trying to have something nice with dynamic props, but it turns out it's not possible today - I would need to nest dynamic drop-downs and objects and arrays within an array and ActivePieces does not allow it
I ranted a bit on our shared Slack channel 😬 

Bottom line: either we ship the action with a several JSON fields (`inputs` and `steps` at least), or we drop it and leave users with custom api call.
I think it's still better to have the explicit action, even if not perfect - WDYT?

```
{
    "job_name": "My Linkedin Company Search job ", -> ShortText, easy
    "unstructured_meta": true, -> Checkbox but effect of "true" is not documented
    "inputs": [ -> Array but why? Should be a single object :face_palm: Can't have nested object in array yet
        { 
            "linkedin_company_search_url": "https://www.linkedin.com/search/results/companies/mySearch"        
        }
    ],
    "steps": [{ -> this gets wild
        "accounts": ["40a0b02-1d2e-406e-a2b7-ad3d83948264"], // A Linkedin account UID - but there is no way to tell from the API that for this specific STEP of this specific WORKFLOW you need a LINKEDIN account and not KASPR or LUSHA or GITHUB or any of the 29 available integrations
        "parameters": {"max_results": 42}, // Can't nest an Object in an Array
        "step_uid": "f34de6f7-da13-4dc2-bfdb-27601e651be4", // By wrapping the Array within a DynamicProperties I can use a StaticDropdown here
    }],
}
```

Reply:
Okay. Let's merge it.